### ISSUE TITLE
TypeError: Cannot read property 'settings' of null

### ISSUE THREAD

**Issue by svennidal** [sindresorhus/speed-test#36]
I'm getting this error: "TypeError: Cannot read property 'settings' of null"

Haven't had any problems before.

**Comment by jwiebalk** [comment]
Changing line 480 in `/usr/local/lib/node_modules/speed-test/node_modules/speedtest-net/index.js`

From:
`serversUrl = 'http://www.speedtest.net/speedtest-servers.php'`
To:
`serversUrl = 'http://c.speedtest.net/speedtest-servers-static.php?load=1'`

Allowed the test to proceed. I'm not sure if something changed on speedtest.net's side.

**Comment by jwiebalk** [comment]
Full error output:

```

      Ping ⠹ 
  Download   
    Upload   

events.js:72
        throw er; // Unhandled 'error' event
              ^
TypeError: Cannot read property 'settings' of null
    at gotServers (/usr/local/lib/node_modules/speed-test/node_modules/speedtest-net/index.js:522:20)
    at /usr/local/lib/node_modules/speed-test/node_modules/speedtest-net/index.js:48:16
    at /usr/local/lib/node_modules/speed-test/node_modules/speedtest-net/index.js:238:7
    at Parser.<anonymous> (/usr/local/lib/node_modules/speed-test/node_modules/speedtest-net/node_modules/xml2js/lib/xml2js.js:489:18)
    at Parser.emit (events.js:95:17)
    at Parser.exports.Parser.Parser.parseString (/usr/local/lib/node_modules/speed-test/node_modules/speedtest-net/node_modules/xml2js/lib/xml2js.js:499:16)
    at Parser.parseString (/usr/local/lib/node_modules/speed-test/node_modules/speedtest-net/node_modules/xml2js/lib/xml2js.js:7:59)
    at exports.parseString (/usr/local/lib/node_modules/speed-test/node_modules/speedtest-net/node_modules/xml2js/lib/xml2js.js:540:19)
    at /usr/local/lib/node_modules/speed-test/node_modules/speedtest-net/index.js:236:5
    at /usr/local/lib/node_modules/speed-test/node_modules/speedtest-net/index.js:48:16
```

**Comment by sindresorhus** [comment]
Can you open an issue on https://github.com/ddsol/speedtest.net instead, which is where the issue is occuring and share the link to the issue here.

**Comment by Skowt** [comment]
Also seeing this issue as of a few days ago. Fixed by following @jwiebalk's solution.

**Comment by lasseebert** [comment]
This is fixed in speedtest.net here: https://github.com/ddsol/speedtest.net/commit/197416347fa055a781a406df9eeba0de528abdba and released in version 1.2.7. Just upgrading that package will make it work again.

**Comment by lvarayut** [comment]
I updated `speedtest.net` to 1.2.7, still got the same error.
